index.md

```diff
@@ -722,38 +758,18 @@ Let's put the sixth item in `wikiCatNameTables` into an object and see what we g
 
 ```r
 famousCats <- wikiCatNameTables[[5]]
+```
+
+```
+#> Error in wikiCatNameTables[[5]]: subscript out of bounds
+```
+
+```r
 head(famousCats)
 ```
 
 ```
-#>   Source VPI 2010[20] VPI 2009[26] VPI 2008[4] VPI 2006[27] YouPet.com[28]
-#> 1      1          Max          Max         Max          Max          Tiger
-#> 2      2        Chloe        Chloe       Chloe       Tigger         Smokey
-#> 3      3        Bella       Tigger        Lucy       Smokey          Kitty
-#> 4      4       Oliver        Tiger      Tigger        Tiger         Shadow
-#> 5      5        Tiger         Lucy       Tiger        Chloe         Tigger
-#> 6      6       Smokey       Smokey      Smokey       Shadow           Baby
-#>   BabyNames.com (male)[29] BabyNames.com (female)[29] BowWow.com[30]
-#> 1                      Max                      Chloe         Tigger
-#> 2                   Tigger                       Lucy          Tiger
-#> 3                    Tiger                      Molly            Max
-#> 4                   Smokey                      Bella         Smokey
-#> 5                   Oliver                     Sophie            Sam
-#> 6                    Buddy                   Princess          Kitty
-#>   Washington Post[31] PetFinder.com (Male)[32] PetFinder.com (Female)[32]
-#> 1               Kitty                      Max                      Molly
-#> 2               Tiger                  Charlie                      Angel
-#> 3                 Max                    Simon                       Lucy
-#> 4              Smokey                     Jack                   Princess
-#> 5              Tigger                    Sammy                      Chloe
-#> 6             Patches                                                    
-#>   Gallup 1990[33] Cook County[34] Sun Times[35]
-#> 1            Baby           Tiger         Kitty
-#> 2         Blackie           Kitty         Tiger
-#> 3        Samantha          Smokey        Smokey
-#> 4             Tom             Max           Sam
-#> 5           Tiger          Shadow         Misty
-#> 6          Casper          Tigger      Samantha
+#> Error in head(famousCats): object 'famousCats' not found
```

Comment:
hmm, looks like `famousCats` is missing?

Reply:
see below.